lib/web_ui/lib/src/engine/js_interop/js_app.dart

```diff
@@ -39,9 +39,9 @@ abstract class FlutterApp {
   }) =>
       FlutterApp._(
         addView: ((JsFlutterViewOptions options) =>
-            futureToPromise(addView(options) as Future<JSAny>)).toJS,
-        removeView: ((int id) =>
-            futureToPromise(removeView(id) as Future<JSObject?>)).toJS,
+            futureToPromise(addView(options).then((int id) => id.toJS))).toJS,
```

Comment:
rather than using a 'then' here, can this be:

```suggestion
            futureToPromise(addView(options) as Future<JSNumber>).toJS,
```
?

Reply:
I haven't tried this, but I suspect it won't work. In wasm mode, we need to actually call `.toJS` to convert to the right type.

Reply:
This js interop fix was done in another PR, so we don't need it here anymore.